samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/ItemAddListener.java

```diff
@@ -893,14 +896,9 @@ else if (itemauthor.getTarget() != null && (itemauthor.getTarget().equals("samba
       }
       else {
         // Came from Assessment Authoring
-
-        AssessmentService assessdelegate;
-        if (isEditPendingAssessmentFlow) {
-        	assessdelegate = new AssessmentService();
-        }
-        else {
-        	assessdelegate = new PublishedAssessmentService();
-        }
+    	  
+    	  if(assessdelegate == null) assessdelegate = isEditPendingAssessmentFlow ? new AssessmentService() : new PublishedAssessmentService();
```

Comment:
It looks like there's some whitespace changes in this file. Personally, I'm fine with this because I know how messed up these files in Samigo can be. However, there are a bunch of single line conditionals being introduced or touched by this patch.

I know single line conditionals look much prettier without the braces, but it can become a readability/maintenance issue down the road. Please add braces to single line control statements.